src/ariel-os-debug/src/lib.rs

```diff
@@ -80,32 +84,27 @@ mod backend {
 
         #[cfg(feature = "defmt")]
         {
-            use rtt_target::ChannelMode::{NoBlockSkip, NoBlockTrim};
+            use rtt_target::ChannelMode::NoBlockSkip;
+            const DEFMT_BUFFER_SIZE: usize = 1024;
```

Comment:
ups, pulling out `DEFMT_BUFFER_SIZE` is an unrelated change. Unless it bothers you my dear reviewer, lets keep it. I'll soon open a PR that makes this configurable.